kimchi/build.rs

```diff
@@ -1,15 +1,13 @@
 // Build script
 
-use std::process::Command;
+use cargo_spec::build::{self, OutputFormat};
 
 fn main() {
     // Rebuild specification
-    assert!(
-        Command::new("make")
-            .args(["-C", "../book/specifications/kimchi", "build"])
-            .status()
-            .expect("failed to get status")
-            .success(),
-        "failed to generate specification markdown"
-    );
+    build::build(
+        "../book/specifications/kimchi/Specification.toml".into(),
```

Comment:
is this going to work always? I think the `../` is not well specified and it's better to use an absolute path from a [cargo env var](https://doc.rust-lang.org/cargo/reference/environment-variables.html), like so: 

```rust
let out_dir env::var("CARGO_MANIFEST_DIR").unwrap();
let spec_file = PathBuf::new(out_dir).join("../book/specifications/kimchi/Specification.toml");
```

at least, I think this should work despite not necessarily pointing to the [workspace](https://github.com/rust-lang/cargo/issues/3946#issuecomment-1402464796)

Reply:
(same for the output bellow)

Reply:
Again, this was not the final PR, just a start.  Thanks for tip 👍🏼